File: frontend/src/api/values/MoovieListTypes.js
const MoovieListTypes = {
    MOOVIE_LIST_TYPE_STANDARD_PUBLIC: { type: 1, description: "Listas que crea un usuario y son públicas" },
    MOOVIE_LIST_TYPE_STANDARD_PRIVATE: { type: 2, description: "Listas que crea un usuario y puso privada" },
    MOOVIE_LIST_TYPE_DEFAULT_PUBLIC: { type: 3, description: "Listas creadas automáticamente por ej: 'Top 50'" },
    MOOVIE_LIST_TYPE_DEFAULT_PRIVATE: { type: 4, description: "Listas creadas automáticamente por ej: 'Watchlist'" },

    getType(typeKey) {
        return this[typeKey]?.type ?? null;
    },
};

export default MoovieListTypes;

File: models/src/main/java/ar/edu/itba/paw/models/MoovieList/MoovieListTypes.java
package ar.edu.itba.paw.models.MoovieList;

public enum MoovieListTypes {
    MOOVIE_LIST_TYPE_STANDARD_PUBLIC(1), //Listas que crea un usuario y son publicas
    MOOVIE_LIST_TYPE_STANDARD_PRIVATE(2), //Listas que crea un usuario y puso privada
    MOOVIE_LIST_TYPE_DEFAULT_PUBLIC(3), //Listas creadas automaticamente por ej: "Top 50"
    MOOVIE_LIST_TYPE_DEFAULT_PRIVATE(4); //Listas creadas automaticamente por ej: "Watchlist"

    private final int type;
    MoovieListTypes(int size) {
        this.type = size;
    }

    public int getType() {
        return type;
    }
}

